src/manifolds/ProbabilitySimplex.jl

```diff
@@ -281,6 +281,45 @@ mean(::ProbabilitySimplex, ::Any...)
 
 default_estimation_method(::ProbabilitySimplex, ::typeof(mean)) = GeodesicInterpolation()
 
+@doc raw"""
+    rand(::ProbabilitySimplex; vector_at=nothing, σ::Real=1.0)
+
+
+When `vector_at` is `nothing`, return a random (uniform) point `x` on the [`ProbabilitySimplex`](@ref)
+manifold `M` by normalizing a sample from an exponential distribution to unit sum.
+
+When `vector_at` is not `nothing`, return a (Gaussian) random vector from the tangent space
+``T_{p}\mathrm{\Delta}^n``by shifting a multivariate Gaussian with standard deviation `σ` 
+to have a zero component sum, see [^ Devroye1986], Theorems 2.1 and 2.2 p. 207, 208.
```

Comment:
```suggestion
to have a zero component sum, see [^Devroye1986], Theorems 2.1 and 2.2 on p. 207 and 208, respectively.
```

we can not push the docs PR (since the PR is not from this repo) but I saw the error in my type which is fixed here.